man/systemctl.xml

```diff
@@ -1863,6 +1863,18 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
             sleep operation is successfully enqueued. It will not wait for the sleep/wake-up cycle to complete.</para>
           </listitem>
         </varlistentry>
+
+        <varlistentry>
```

Comment:
I don't think this option is relevant enough to be part of systemctl. Instead, please move it to systemd-analyze, where I think it fits much better, along with "set-log-level" and "set-log-target" and friends.

(Hmm, that said, I think "systemctl set-log-level" plus "systemctl get-log-level" aren't actually a particularly nice interface though. I think it should have been "systemctl log-level" (i.e. without the "get-" or "set-" suffix) and then decide whether to get or set the property depending on whether there's something else specified. Doing it that way works against the proliferation of systemd-analyze commands. Long story short, I think there should be "systemd-analyze runtime-watchdog-sec" which will set the watchdog freq if you specify one more argument, and get it if not.)

Reply:
Also, what shall happen with the effect of this when the user does "systemctl daemon-reexec"? there should be some logic for that.